maat-court-data-api/src/main/java/gov/uk/courtdata/reporder/controller/RepOrderController.java

```diff
@@ -81,4 +84,15 @@ public ResponseEntity<Object> getRepOrderMvoByRepIdAndVehicleOwner(@PathVariable
 
     }
 
+    @GetMapping(value = {"/{repId}/rep-order-count-with-sentence-order-date"}, produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(description = "Retrieve a rep order record")
```

Comment:
Description is misleading, should this be Retrieve rep order count for a given sentence order date?

Reply:
Thanks. Done now